### PR TITLE
Split tests

### PR DESCRIPTION
**Pull request by litman90**
This PR split the unitary and regtests from the "examples" tests.

The idea is that we trigger  unitary and regtests in each push and
all the test when doing a PR to master.

### CODE REVIEW

**Comment by litman90** [comment]
I also simplified the regression tests for the instanton functionality because they were taking an unnecessarily long time.

File: bin/i-pi-tests [hunk 1]
```diff
@@ -0,0 +1 @@
+../tools/py/tests.py
```
Comment (mahrossi) [inline bin/i-pi-tests:1]: is this the most general way to link these files? why not making it a link to tests.py instead?

Comment (litman90) [reply bin/i-pi-tests:1]: I did a symbolic link  (ln -s ) like it is done for all the files in the bin folder except for "i-pi"
What are you suggesting?

Comment (mahrossi) [reply bin/i-pi-tests:1]: ah is this how a symbolic link appears in github? interesting, I did not know! sorry, I only looked at the review interface and not the command line. then you did what I suggested.

**Review by mahrossi — APPROVED** [review]

**Comment by eszterpos** [comment]
This is great, and will make the pushes considerably a smoother experience for the developers.

**Pull request merged**